go/oasis-node/cmd/debug/txsource/workload/oversized.go

```diff
@@ -10,37 +10,45 @@ import (
 
 	"github.com/oasisprotocol/oasis-core/go/common/crypto/signature"
 	memorySigner "github.com/oasisprotocol/oasis-core/go/common/crypto/signature/signers/memory"
-	"github.com/oasisprotocol/oasis-core/go/common/logging"
 	consensus "github.com/oasisprotocol/oasis-core/go/consensus/api"
 	"github.com/oasisprotocol/oasis-core/go/consensus/api/transaction"
 	consensusGenesis "github.com/oasisprotocol/oasis-core/go/consensus/genesis"
 	staking "github.com/oasisprotocol/oasis-core/go/staking/api"
 )
 
-const (
-	// NameOversized is the name of the oversized workload.
-	NameOversized = "oversized"
+// NameOversized is the name of the oversized workload.
+const NameOversized = "oversized"
```

Comment:
I think this part

https://github.com/oasisprotocol/oasis-core/blob/b59d6b3734f45e63b6da497746c386269b51b2ab/go/oasis-node/cmd/debug/txsource/workload/oversized.go#L127-L131

should now be removed from the workload as the disable checktx flag was removed.

Reply:
Done.